src/main/java/com/epam/spring/homework2/beans/MyBean.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.spring.homework2.beans;
+
+public class MyBean implements MyValidator {
+    protected String name;
+    protected int value;
+
+    public MyBean(String name, int value) {
+        this.name = name;
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return getClass().getSimpleName() +
+                "name='" + name + '\'' +
+                ", value=" + value +
+                '}';
+    }
+
+    @Override
+    public void validate() {
```

Comment:
That's very good that you've moved common validation logic to separate class, good job!